src/plugins/lvm-dbus.c

```diff
@@ -3369,6 +3375,130 @@ gboolean bd_lvm_cache_create_cached_lv (const gchar *vg_name, const gchar *lv_na
     return TRUE;
 }
 
+/**
+ * bd_lvm_writecache_attach:
+ * @vg_name: name of the VG containing the @data_lv and the @cache_pool_lv LVs
+ * @data_lv: data LV to attache the @cache_pool_lv to
+ * @cache_lv: cache (fast) LV to attach to the @data_lv
+ * @extra: (allow-none) (array zero-terminated=1): extra options for the cache attachment
+ *                                                 (just passed to LVM as is)
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether the @cache_lv was successfully attached to the @data_lv or not
+ *
+ * Tech category: %BD_LVM_TECH_WRITECACHE-%BD_LVM_TECH_MODE_MODIFY
+ */
+gboolean bd_lvm_writecache_attach (const gchar *vg_name, const gchar *data_lv, const gchar *cache_lv, const BDExtraArg **extra, GError **error) {
+    GVariantBuilder builder;
+    GVariant *params = NULL;
+    gchar *lv_id = NULL;
+    gchar *lv_obj_path = NULL;
+    gboolean success = FALSE;
+
+    /* both LVs need to be inactive for the writecache convert to work */
```

Comment:
I think this fact is worth mentioning in the function docs.

Reply:
Ok, I've added a note to the docstring.